src/main/java/com/lucidworks/spark/query/SerializedTuple.java

```diff
@@ -0,0 +1,82 @@
+package com.lucidworks.spark.query;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.solr.client.solrj.io.Tuple;
+
+@SuppressWarnings({ "unchecked", "rawtypes" })
+public class SerializedTuple implements Serializable {
+
```

Comment:
The original implementation of SolrStreamIterator worked with Tuples from the underlying SolrStream (which is a TupleStream). But then I saw in that the SolrRDD works with SolrDocuments. So converted it to that. This file is not used anymore. I will remove it.